lib/pbench/agent/results.py

```diff
@@ -103,9 +99,7 @@ def make_result_tb(self):
                 tarball.unlink()
             except Exception as exc:
                 if exc.errno != errno.ENOENT:
-                    raise OSError(
-                        "error removing failed tar ball, '{}', {}", tarball, exc
-                    )
+                    raise OSError(f"error removing failed tar ball, '{tarball}', {exc}")
```

Comment:
In other exception text, you're capitalizing the message as a diagnostic sentence to the user; this should be "Error removing..." for consistency.

Reply:
Done